sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -404,6 +404,7 @@ export const Constants: {
     };
     WritableLocations: string;
     ReadableLocations: string;
+    LocationUnavailableExpirationTime: number;
```

Comment:
nit: Please qualify the naming with units (ex: InMs etc...)

Reply:
addressed